data/json/items/gun/223.json

```
@@ -1313,5 +1313,75 @@
       }
     ],
     "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "rdb_223",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "Bullpup rifle 5.56x45" },
+    "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the AR-15 platform and a 5.56x45 cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rdb_223",
+        "name": { "str_sp": "Kel-Tec RDB" },
+        "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the AR-15 platform and a 5.56 NATO cartridge.",
```

```suggestion
        "description": "A semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the AR-15 platform and a 5.56 NATO cartridge.",
```

You can use a more elaborate description on the non-generic you know 😛

---

i used this as an example for "This" or "A".
https://github.com/CleverRaven/Cataclysm-DDA/pull/65918/files/db0c147abb8bfd5ddca961741c3314dae3738167#diff-71f0f2b958b18cee00872ca2b81173dd3c98c68ec22b7cc595eb93a3c648e11f